corehq/apps/app_manager/views/download.py

```diff
@@ -315,7 +315,7 @@ def download_index(request, domain, app_id, template="app_manager/download_index
     all the resource files that will end up zipped into the jar.
 
     """
-    files = None
+    files = []
```

Comment:
This is an unrelated change. Prevents the page from 500ing if the try block fails.